Makefile

```diff
@@ -76,67 +77,80 @@ build/compiler-rt.BUILT: build/llvm.BUILT
 		-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON \
 		$(LLVM_PROJ_DIR)/compiler-rt/lib/builtins
 	ninja $(NINJA_FLAGS) -v -C build/compiler-rt install
+	# Install clang-provided headers.
 	cp -R $(ROOT_DIR)/build/llvm/lib/clang $(PREFIX)/lib/
 	touch build/compiler-rt.BUILT
 
+# Flags for libcxx.
+LIBCXX_CMAKE_FLAGS = \
+    -DCMAKE_TOOLCHAIN_FILE=$(ROOT_DIR)/wasi-sdk.cmake \
+    -DLLVM_CONFIG_PATH=$(ROOT_DIR)/build/llvm/bin/llvm-config \
+    -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON \
+    -DLIBCXX_ENABLE_THREADS:BOOL=OFF \
+    -DLIBCXX_HAS_PTHREAD_API:BOOL=OFF \
+    -DLIBCXX_HAS_EXTERNAL_THREAD_API:BOOL=OFF \
+    -DLIBCXX_BUILD_EXTERNAL_THREAD_LIBRARY:BOOL=OFF \
+    -DLIBCXX_HAS_WIN32_THREAD_API:BOOL=OFF \
+    -DCMAKE_BUILD_TYPE=RelWithDebugInfo \
+    -DLIBCXX_ENABLE_SHARED:BOOL=OFF \
+    -DLIBCXX_ENABLE_EXPERIMENTAL_LIBRARY:BOOL=OFF \
+    -DLIBCXX_ENABLE_EXCEPTIONS:BOOL=OFF \
+    -DLIBCXX_ENABLE_FILESYSTEM:BOOL=OFF \
+    -DLIBCXX_CXX_ABI=libcxxabi \
+    -DLIBCXX_CXX_ABI_INCLUDE_PATHS=$(LLVM_PROJ_DIR)/libcxxabi/include \
+    -DLIBCXX_HAS_MUSL_LIBC:BOOL=ON \
+    -DLIBCXX_ABI_VERSION=2 \
+    -DWASI_SDK_PREFIX=$(PREFIX) \
+    --debug-trycompile
+
 build/libcxx.BUILT: build/llvm.BUILT build/compiler-rt.BUILT build/wasi-libc.BUILT
+	# Do the build.
 	mkdir -p build/libcxx
-	cd build/libcxx; cmake -G Ninja \
-		-DCMAKE_TOOLCHAIN_FILE=$(ROOT_DIR)/wasi-sdk.cmake \
-		-DLLVM_CONFIG_PATH=$(ROOT_DIR)/build/llvm/bin/llvm-config \
-		-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON \
-		-DLIBCXX_ENABLE_THREADS:BOOL=OFF \
-		-DLIBCXX_HAS_PTHREAD_API:BOOL=OFF \
-		-DLIBCXX_HAS_EXTERNAL_THREAD_API:BOOL=OFF \
-		-DLIBCXX_BUILD_EXTERNAL_THREAD_LIBRARY:BOOL=OFF \
-		-DLIBCXX_HAS_WIN32_THREAD_API:BOOL=OFF \
-		-DCMAKE_BUILD_TYPE=RelWithDebugInfo \
-		-DLIBCXX_ENABLE_SHARED:BOOL=OFF \
-		-DLIBCXX_ENABLE_EXPERIMENTAL_LIBRARY:BOOL=OFF \
-		-DLIBCXX_ENABLE_EXCEPTIONS:BOOL=OFF \
-		-DLIBCXX_ENABLE_FILESYSTEM:BOOL=OFF \
-		-DLIBCXX_CXX_ABI=libcxxabi \
-		-DLIBCXX_CXX_ABI_INCLUDE_PATHS=$(LLVM_PROJ_DIR)/libcxxabi/include \
-		-DLIBCXX_HAS_MUSL_LIBC:BOOL=ON \
-		-DLIBCXX_ABI_VERSION=2 \
-		-DLIBCXX_LIBDIR_SUFFIX=/wasm32-wasi \
-		-DWASI_SDK_PREFIX=$(PREFIX) \
-		-DCMAKE_C_FLAGS="$(DEBUG_PREFIX_MAP)" \
-		-DCMAKE_CXX_FLAGS="$(DEBUG_PREFIX_MAP)" \
-		--debug-trycompile \
-		$(LLVM_PROJ_DIR)/libcxx
+	cmake -B build/libcxx -G Ninja $(LIBCXX_CMAKE_FLAGS) \
+	    -DCMAKE_C_FLAGS="$(DEBUG_PREFIX_MAP)" \
+	    -DCMAKE_CXX_FLAGS="$(DEBUG_PREFIX_MAP)" \
+            -DLIBCXX_LIBDIR_SUFFIX=/wasm32-wasi \
```

Comment:
indentation

Reply:
My editor had inserted 8 spaces instead of a tab :-/. Fixed now.